pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>fr.cnes.sonarqube.plugins</groupId>
     <artifactId>sonar-hadolint-plugin</artifactId>
     <packaging>sonar-plugin</packaging>
-    <version>1.0.0</version>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
Is it intentional to you to append `-SNAPSHOT` ?

Reply:
Well, that was put during the development phase, and I obviously forgot to remove this suffix.
I'll fix that.